spring-cloud-vault-config/src/main/java/org/springframework/cloud/vault/config/VaultProperties.java

```diff
@@ -1287,4 +1297,65 @@ public void setExpiryThreshold(Duration expiryThreshold) {
 
 	}
 
+	/**
+	 * Vault retry configuration
+	 *
+	 * @since 3.0.2
+	 */
+	public static class Retry {
```

Comment:
Let's mount retry properties externally, similar to https://github.com/spring-cloud/spring-cloud-config/blob/master/spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/RetryProperties.java.

Reply:
Implemented in updated changes.